src/Pages/Shop/Item/Item.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { priceFormat } from '../../../utils/priceFormat';
+import './Item.scss';
+
+class Item extends React.Component {
+  render() {
+    const { id, price, koreanName, englishName, image } = this.props;
+
+    return (
+      <li className="item">
+        <Link to={`/shop/detail/${id}`}>
+          <div className="image-container">
+            <img
+              alt={koreanName}
+              src={image === null ? '/images/defaultImage.jpg' : image}
```

Comment:
```suggestion
              src={image ? image : '/images/defaultImage.jpg'}
```
값이 있을 때는 보여주고, 없을 때는 기본 이미지를 보여준다 쪽이 흐름상 자연스러워 보입니다

Reply:
네 수정하겠습니다